### ISSUE TITLE
[Bug]: Video scan on certain videos in latest beta not working

### ISSUE THREAD

**Issue by TwoPercentSkill** [ThioJoe/YT-Spammer-Purge#654]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

Well, I was trying to scan all the comments on one of PewDiePie's videos but then an error occurred in the middle of scanning the comments. After retrying the scan on the same video the error occurred again after scanning almost the same amount of comments. After that I tried some other videos. It worked as I would expect on some videos like "https://www.youtube.com/watch?v=k5QXtP8VP7A" but other videos like "https://www.youtube.com/watch?v=E1zLaK1TH54" result in the previously mentioned error.

### Release version

v2.16.0-Beta2

### Steps to reproduce

1. Use version v2.16.0-Beta2
2. Scan the video https://www.youtube.com/watch?v=pTdjcLlElD4 in auto-smart mode.
3. After scanning through about 3100 comments this error occurs.

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
Traceback (most recent call last):ents Scanned: 3099 | Replies Scanned: 556 | Matches Found So Far: 89
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1670, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1647, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1209, in primaryInstance
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1192, in scan_video
  File "Scripts\operations.py", line 120, in get_comments
  File "Scripts\operations.py", line 910, in check_against_filter
  File "Scripts\operations.py", line 844, in findObf
AttributeError: 'list' object has no attribute 'lower'
------------------------------------------------
Error Message: 'list' object has no attribute 'lower'

Unknown Error - Code: Z-3 occurred. If this keeps happening, consider posting a bug report
on the GitHub issues page, and include the above error info.
Short Link: TJoe.io/bug-report
```

**Comment by Firecul** [comment]
Confirmed (Using your links)

![WindowsTerminal_Windows_PowerShell (28)](https://user-images.githubusercontent.com/34950371/153970992-ebfb7348-60ce-4386-9bee-d698cba012b4.png)
![WindowsTerminal_Windows_PowerShell (27)](https://user-images.githubusercontent.com/34950371/153970994-004ad91e-7bf4-422b-9b08-8030d0d0e42a.png)

**Comment by ThioJoe** [comment]
Should be fixed in the new release, 2.16.0-Beta3
